tools/packaging/release/update-repository-version.sh

```diff
@@ -235,6 +248,34 @@ EOT
 		out=""
 		out=$("${hub_bin}" pull-request -b "${target_branch}" -F "${notes_file}" 2>&1) || echo "$out" | grep "A pull request already exists"
 	fi
+
+	if [ "${repo}" == "kata-containers" ] && [ "${target_branch}" == "main" ] && [[ "${new_version}" =~ "rc" ]]; then
+		reverting_kata_deploy_changes_branch="revert-kata-deploy-changes-after-${new_version}-release"
+		git checkout -b "${reverting_kata_deploy_changes_branch}"
+
+		git revert --no-commit ${kata_deploy_commit} >>/dev/null
```

Comment:
I think this revert block of instructions is error-prone. Suppose you change the kata-deploy in the future, new files are added (others deleted) and you need to revert changes on them; so you will need to adjust the list of `git add` below.

If I understood correctly you want to revert the previous commit fully, so I propose to do the following instead:
```
info "Creating the commit reverting the kata-deploy changes"
git revert --no-edit ${kata_deploy_commit} >>/dev/null
commit_msg="$(generate_revert_kata_deploy_commit $new_version)"
git commit --amend -m "${commit_msg}"
```

Reply:
Brilliant, this simplifies things, indeed.